controllers/v1/userCourses.js

```
@@ -0,0 +1,148 @@
+			try {
+				let userCoursesData = await userCoursesHelper.syncUserCourse(req.body, req.userDetails)
+				userCoursesData['result'] = userCoursesData.data
+				return resolve(userCoursesData)
+			} catch (error) {
+				return reject({
+					status: error.status || HTTP_STATUS_CODE.internal_server_error.status,
+					message: error.message || HTTP_STATUS_CODE.internal_server_error.message,
+					errorObject: error,
+				})
+			}
+		})
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)